apps/web/playwright/fixtures/regularBookings.ts

```diff
@@ -376,16 +381,107 @@ export function createBookingPageFixture(page: Page) {
       const confirmButton = options.isReschedule ? "confirm-reschedule-button" : "confirm-book-button";
       await fillAllQuestions(eventTypePage, questions, options);
       await eventTypePage.getByTestId(confirmButton).click();
-      await eventTypePage.waitForTimeout(400);
-      if (await eventTypePage.getByRole("heading", { name: "Could not book the meeting." }).isVisible()) {
-        await eventTypePage.getByTestId("back").click();
-        await eventTypePage.getByTestId("time").last().click();
-        await fillAllQuestions(eventTypePage, questions, options);
-        await eventTypePage.getByTestId(confirmButton).click();
-      }
       const scheduleSuccessfullyPage = eventTypePage.getByText(scheduleSuccessfullyText);
       await scheduleSuccessfullyPage.waitFor({ state: "visible" });
       await expect(scheduleSuccessfullyPage).toBeVisible();
     },
+    addGuests: async (eventTypePage: Page, options: { guests: string[] }) => {
+      await eventTypePage.getByTestId("add-guests").click();
+      for (const guest of options.guests) {
+        await eventTypePage.getByPlaceholder("Email").fill(guest);
+      }
+    },
+    createTeam: async (name: string) => {
+      const teamsText = (await localize("en"))("teams");
```

Comment:
Why are we needing to use translated text instead of IDs to find these elements?

Reply:
We were trying to reduce the changes, but makes sense! We already implemented it, pushing now 😄